porter/v1/cluster_control_plane.proto

```diff
@@ -838,6 +839,33 @@ enum EnumServiceDeploymentStatus {
 // UpdateServiceDeploymentStatusResponse is the response object for updating the deployment status of a service
 message UpdateServiceDeploymentStatusResponse {}
 
+message SetupExternalSecretsRequest {
```

Comment:
We seem to be calling this "env group type" elsewhere instead of external secrets.
Would it make more sense to change both to `SetupEnvGroupProvider` and `EnvGroupProvider`?